psi4/driver/p4util/solvers.py

```diff
@@ -126,8 +124,8 @@ def cg_solver(rhs_vec, hx_function, preconditioner, guess=None, printer=None, pr
         resid = printer(0, x_vec, r_vec)
     elif printlvl:
         # core.print_out('         CG Iteration Guess:    Rel. RMS = %1.5e\n' %  np.mean(resid))
-        core.print_out("    %5s %14.3e %12.3e %7d %9d\n" % ("Guess", np.mean(resid), np.max(resid), len(z_vec),
-                                                            time.time() - tstart))
+        core.print_out("    %5s %14.3e %12.3e %7d %9d\n" %
```

Comment:
@loriab Can we use strings yet?

Reply:
f-strings? yes.

only f-string debugging (https://tirkarthi.github.io/programming/2019/05/08/f-string-debugging.html) like `f"{resid=}` is forbidden to us.

Reply:
I'm moving printing to use f-strings while a move through the code.